packages/scene-composer/src/components/wrappers/ReactMarkdownWrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+import ReactMarkdown from 'react-markdown';
+
+import './ReactMarkdownWrapper.scss';
+
+export interface ReactMarkdownWrapperProps {
+  content: string;
+  className?: string;
+}
+export const ReactMarkdownWrapper: FC<ReactMarkdownWrapperProps> = ({ content, className }) => {
```

Comment:
What's the significance of this component? Is this something can be reused in the future?